config/initializers/strong_migrations.rb

```diff
@@ -0,0 +1,5 @@
+# https://github.com/ankane/strong_migrations#existing-migrations
+StrongMigrations.start_after = 2019_12_27_114543
```

Comment:
While it may look unusual, this is the way the timestamp gets formatted in `db/schema.rb`, so I thought it's easiest to keep it the same way.